test/Directory.Build.props

```diff
@@ -1,5 +1,8 @@
 <Project>
 	<Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
+	<PropertyGroup>
+		<TargetFrameworks>netcoreapp3.0;netcoreapp3.1</TargetFrameworks>
```

Comment:
If we are targetting net standard I think we should run the tests from net48, going to make setting up the test matrix more interesting tho!

Reply:
As discussed offline, .net "standard" 2.1 does not support .net framework: https://docs.microsoft.com/en-us/dotnet/standard/net-standard#net-implementation-support